tests/supervisor/test_execution.py

```diff
@@ -73,29 +80,34 @@ async def test_create_execution_online(vm_hash: ItemHash = None):
     engine = metrics.setup_engine()
     await metrics.create_tables(engine)
 
-    message = await get_message(ref=vm_hash)
+    message, original_message = await load_updated_message(vm_hash)
 
     execution = VmExecution(
         vm_hash=vm_hash,
         message=message.content,
-        original=message.content,
+        original=original_message.content,
         snapshot_manager=None,
         systemd_manager=None,
         persistent=False,
     )
 
-    # Downloading the resources required may take some time, limit it to 10 seconds
-    await asyncio.wait_for(execution.prepare(), timeout=30)
+    # Downloading the resources required may take some time, limit it to 120 seconds
+    # it's a bit slower in github ci
+    await asyncio.wait_for(execution.prepare(), timeout=120)
 
     vm = execution.create(vm_id=3, tap_interface=None)
+
     # Test that the VM is created correctly. It is not started yet.
     assert isinstance(vm, AlephFirecrackerProgram)
+    vm.enable_console = True
+    vm.fvm.enable_log = True
     assert vm.vm_id == 3
 
     await execution.start()
     await execution.stop()
 
 
+# This test depends on having a vm-connector running on port 4021
```

Comment:
Should the test assert that the `vm-connector` can be reached ?

Reply:
Won't really change anything